src/header/header.scss

```diff
@@ -245,15 +268,104 @@
       &:hover {
         svg.iui-header-button-icon {
           @include themed {
-            background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+            background-color: rgba(
+              t(iui-color-foreground-primary-rgb),
+              t(iui-opacity-6)
+            );
           }
         }
       }
+
+      &:focus,
+      &:focus-visible {
+        @include themed {
+          box-shadow: t(iui-focus-box-shadow),
+            0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
+
+      &,
+      &:focus:not(:focus-visible) {
+        @include themed {
+          box-shadow: 0 $iui-xxs 0 0 t(iui-color-foreground-primary);
+        }
+      }
+    }
+  }
+
+  .iui-button-split-menu {
+    height: 100%;
```

Comment:
![image](https://user-images.githubusercontent.com/81580355/127847069-ed96646a-da96-469f-bcaf-d66397a1661c.png)
found this generated...not sure where duplication comes from..but it is in our split button. Will need to investigate

Reply:
This comes from `buttons/classes.scss`.
![image](https://user-images.githubusercontent.com/9084735/128512027-763904f5-3b8b-4c87-9e0f-7b572494e3c5.png)

Need to remove line 35 because we already include button-group mixin in split-menu mixin.

Can tackle in #225